pkgs/servers/web-apps/discourse/default.nix

```diff
@@ -2,20 +2,21 @@
 , fetchFromGitHub, bundlerEnv, callPackage
 
 , ruby, replace, gzip, gnutar, git, cacert, util-linux, gawk
-, imagemagick, optipng, pngquant, libjpeg, jpegoptim, gifsicle, libpsl
-, redis, postgresql, which, brotli, procps, rsync, nodePackages, v8
+, imagemagick, optipng, pngquant, libjpeg, jpegoptim, gifsicle, jhead
+, libpsl, redis, postgresql, which, brotli, procps, rsync
+, nodePackages, v8
```

Comment:
If the inputs would be one per line you would have a very nice diff.

Reply:
Yeah, it it would be, but the list would also be much longer, so it's a trade-off. I don't think the dependencies are updated frequently enough that the slight ugliness is cause for concern. If that changes, however, it could be worth rewriting it.